File: app/src/main/java/ru/job4j/vkfriendskt/di/NetworkServiceModule.java
package ru.job4j.vkfriendskt.di;



import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;
import ru.job4j.vkfriendskt.network.NetworkService;

@Module
public class NetworkServiceModule {
    @Singleton
    @Provides
    public NetworkService providesNetworkService() {
        return new NetworkService();
    }
}

File: app/src/main/java/ru/job4j/vkfriendskt/di/NetworkServiceModule.kt
package ru.job4j.vkfriendskt.di

import dagger.Module
import dagger.Provides
import ru.job4j.vkfriendskt.network.NetworkService
import javax.inject.Singleton

@Module
class NetworkServiceModule {
    @Singleton
    @Provides
    fun providesNetworkService(): NetworkService {
        return NetworkService()
    }
}